presentation/src/main/java/com/keeply/presentation/core/components/InsightTextField.kt

```diff
@@ -0,0 +1,106 @@
+package com.keeply.presentation.core.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.keeply.presentation.core.theme.KeeplyTheme
+import com.keeply.presentation.core.theme.LocalColors
+
+@Composable
+fun InsightTextField(
+    value: String,
+    onValueChange: (String) -> Unit,
+    modifier: Modifier = Modifier,
+    placeholder: String = "",
+    maxLength: Int = 300
+) {
+    Column (
+        modifier = modifier
+            .fillMaxWidth()
+            .background(
+                color = KeeplyTheme.colors.neutral200,
+                shape = RoundedCornerShape(4.dp)
+            )
+            .padding(16.dp)
+    ) {
+        Box {
+            if (value.isEmpty()) {
+                KeeplyText(
+                    text = placeholder,
+                    style = KeeplyTheme.typography.body,
+                    color = LocalColors.current.neutral500
+                )
+            }
+
+            BasicTextField(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .heightIn(
+                        min = 82.dp
+                    ),
+                value = value,
+                onValueChange = { newText ->
+                    if (newText.length <= maxLength) {
+                        onValueChange(newText)
+                    }
+                },
+                textStyle = KeeplyTheme.typography.body,
+                cursorBrush = SolidColor(Color.Black),
+            )
+        }
+
+        KeeplyText(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 4.dp),
```

Comment:
p1:
```suggestion
                .padding(top = 12.dp),
```
cc. https://www.figma.com/design/M4enxYIUhsKxggjiAhEaxn?node-id=1106-13752&m=dev#1342621710
댓글 참고 부탁드립니당!

Reply:
@euzl 수정 반영했어요~ 컨플릭트도 수정했어요